client/state/action-types.js

```diff
@@ -738,6 +738,13 @@ export const REWIND_RESTORE_UPDATE_PROGRESS = 'REWIND_RESTORE_UPDATE_PROGRESS';
 export const REWIND_STATUS_ERROR = 'REWIND_STATUS_ERROR';
 export const REWIND_STATUS_REQUEST = 'REWIND_STATUS_REQUEST';
 export const REWIND_STATUS_UPDATE = 'REWIND_STATUS_UPDATE';
+export const REWIND_BACKUP = 'REWIND_BACKUP';
+export const REWIND_BACKUP_DISMISS = 'REWIND_BACKUP_DISMISS';
+export const REWIND_BACKUP_DISMISS_PROGRESS = 'REWIND_BACKUP_DISMISS_PROGRESS';
+export const REWIND_BACKUP_PROGRESS_REQUEST = 'REWIND_BACKUP_PROGRESS_REQUEST';
+export const REWIND_BACKUP_PLEASE = 'REWIND_BACKUP_PLEASE';
```

Comment:
can we talk about `_PLEASE`? what does it mean? does it mean the same thing as `_REQUEST`? I'm only asking because we have existing patterns in action type names and it might be confusing to see even more pop up, especially when we're trying to trim up how many action types we have.

Reply:
Yeah, it's semantically the same than a _REQUEST. I'll update the names for BACKUP and RESTORE